application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalJwtFactory.java

```diff
@@ -24,7 +24,7 @@ public String createToken(
     Date nowDate = new Date(nowMs);
 
     KeyStoreKeyFactory keyStoreKeyFactory =
```

Comment:
I wonder if we can add these local files to be ignored by the git password hook...

Reply:
git secret -scan doesn't have option to ignore files but we can include files with this option